docs/quiz/quiz.yaml

```diff
@@ -1,5 +1,85 @@
 paths:
   /quizzes:
+    get:
+      security:
+        - bearerAuth: []
+      tags:
+        - Quizzes
+      summary: Find all quizzes.
+      description: Finds and returns an array with a list of all your own quizzes.
+      produces:
+        - application/json
+      parameters:
+        - in: query
+          name: title
+          schema:
+            type: string
+          required: false
+        - in: query
+          name: sort
+          schema:
+            type: string
+          required: false
+        - in: query
+          name: skip
+          schema:
+            type: string
+          required: false
+        - in: query
+          name: limit
+          schema:
+            type: string
+          required: false
+      responses:
+        200:
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/definitions/quizBody'
+              example:
+                items:
+                  - _id: 64ca5914b57f2442403394a5
+                    title: IT
+                    items:
+                      - question: Is it the best programming language?
+                        answers:
+                          - text: Yes,
+                            isCorrect: false
+                          - text: Yes, of course,
+                            isCorrect: true
+                  - _id: 64ca5932b57f2442403394a9
+                    title: Chemistry
+                    items:
+                      - question: What is the chemical symbol for water?
+                        answers:
+                          - text: H2O,
+                            isCorrect: false
+                          - text: O2H
+                            isCorrect: true
+                          - text: H2O2
+                            isCorrect: true
+                count: 2
```

Comment:
I guess, author field should also be here

Reply:
Added